src/Dotnet.Script.Tests/ScriptExecutionTests.cs

```diff
@@ -250,7 +250,9 @@ public void ShouldSupportInlineNugetReferencesWithTrailingSemicoloninEvaluatedCo
 
         [Theory]
         [InlineData("https://gist.githubusercontent.com/seesharper/5d6859509ea8364a1fdf66bbf5b7923d/raw/0a32bac2c3ea807f9379a38e251d93e39c8131cb/HelloWorld.csx",
-                    "Hello World")]
+            "Hello World")]
+        [InlineData("http://gist.githubusercontent.com/seesharper/5d6859509ea8364a1fdf66bbf5b7923d/raw/0a32bac2c3ea807f9379a38e251d93e39c8131cb/HelloWorld.csx",
```

Comment:
Should be `https://`

Reply:
This is intentionally http because this variant tests following a redirect (which would be issued from http to https)